src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorRenameProvider.ts

```diff
@@ -31,7 +31,7 @@ export class RazorRenameProvider
         const projection = await this.getProjection(document, position, token);
         if (!projection || projection.languageKind !== LanguageKind.CSharp) {
             // We only support C# renames for now. Reject the rename request.
-            return Promise.reject('Cannot rename this symbol.');
+            return null;  // Promise.reject('Cannot rename this symbol.');
```

Comment:
Will definitely need to update the comments around this as to why you had to do this 😢